backup/setup/schools.json

```diff
@@ -33,7 +33,7 @@
 				"STUDENT_LIST": true
 			}
 		},
-        "features" : []
+        "features" : ["oauthProvisioningEnabled"]
```

Comment:
Why you must add this to all schools? It is a nbc feature for now.

Reply:
There is OAuth on other instances too. Namely Erwin.

Reply:
Yes but you add this to all schools on all instances. See migration, this can not be correct. The features of the school, can be disbaled on school level from admins. That this one it not mapped atm in the UI is a other thing, but it should. But maybe we must speak about it. It is unclear to me for now.